### PR TITLE
changed flows and moved files according to suggested changes

### PR DESCRIPTION
**Pull request by abhijitpal1247**
- Changes in README.md
- Changes in guardrails-library.md
- Added flows.co and moved the default flow to the file

### CODE REVIEW

File: nemoguardrails/library/autoguard/flows.co
Comment (KimiJL) [inline nemoguardrails/library/autoguard/flows.co]: hmm @rahm-decypher @abhijitpal1247 

Not sure if i'm reading this correctly, my interpretation is that it's checking input prompt for factchecking purposes. Does that make sense?

Comment (rahm-decypher) [reply nemoguardrails/library/autoguard/flows.co]: fact checking should be on the model output and the list of evidence documents. please change the flow if that is not the case

File: nemoguardrails/library/autoguard/README.md
Comment (rahm-decypher) [inline nemoguardrails/library/autoguard/README.md]: i think the matching score is between 0 to 1, not just 0 and 1

File: nemoguardrails/library/autoguard/flows.co
Comment (abhijitpal1247) [reply nemoguardrails/library/autoguard/flows.co]: @rahm-decypher, I have kept factcheck for both input and output end. For example in the demo we have prompts which are used to show fact-checking guardrail.

@KimiJL coming to code, it is in non blocking mode, we can see that the input_result (autoguard response after applying factcheck on the user input) is fetched here: https://github.com/decypher-ai/NeMo-Guardrails/blob/suggested-changes/nemoguardrails/library/autoguard/flows.co#L23
Later after obtaining the factcheck result on bot response, output_result, both the input_result and output_result is being displayed.

Comment (abhijitpal1247) [reply nemoguardrails/library/autoguard/flows.co]: we have two separate functions for user input and bot response respectively. Should I make the factheck flow a blocking one as we have the normal autoguard flow in blocking mode. @KimiJL @rahm-decypher ?

File: nemoguardrails/library/autoguard/README.md
Comment (abhijitpal1247) [reply nemoguardrails/library/autoguard/README.md]: resolved in further commits

File: nemoguardrails/library/autoguard/flows.co
Comment (KimiJL) [reply nemoguardrails/library/autoguard/flows.co]: The idea should be that factcheck isn't run on the user input, and only on the bot response, regardless of blocking or non-blocking

Comment (abhijitpal1247) [reply nemoguardrails/library/autoguard/flows.co]: removed the input factcheck guardrail

**Review by KimiJL — APPROVED** [review]
LGTM, should wait for Rahm's approval as well, and please remember to redo the contribution guides for linting etc.

**Review by rahm-decypher — APPROVED** [review]
lgtm

**Pull request merged**
